docs/en/reference/filters.rst

```diff
@@ -28,10 +28,11 @@ table alias of the SQL table of the entity.
     In the case of joined or single table inheritance, you always get passed the ClassMetadata of the
     inheritance root. This is necessary to avoid edge cases that would break the SQL when applying the filters.
 
-Parameters for the query should be set on the filter object by
-``SQLFilter#setParameter()``. Only parameters set via this function can be used
-in filters.  The ``SQLFilter#getParameter()`` function takes care of the
-proper quoting of parameters.
+For the filter to correctly function the following rules must be followed, failure to do so will lead to unexpected results from the query cache.
```

Comment:
```suggestion
For the filter to correctly function, the following rules must be followed. Failure to do so will lead to unexpected results from the query cache.
```

`following rules must be followed` sounds funny, but I have no suggestion here.

Reply:
"the following rules must be respected" or "the following rules must be observed" ?